miniapps/meshing/mesh-explorer.cpp

```diff
@@ -1190,6 +1191,33 @@ int main (int argc, char *argv[])
          cout << "New VTK mesh file: " << omesh_file << endl;
       }
 
+      if (mk == 'D')
+      {
+         cout << "What type of DataCollection?\n"
+              "p) ParaView Data Collection\n"
+              "v) VisIt Data Collection\n"
+              "--> " << flush;
+         char dk;
+         cin >> dk;
+         if (dk == 'p')
+         {
+            const char omesh_file[] = "mesh-explorer-paraview";
+            ParaViewDataCollection dc(omesh_file, mesh);
+            dc.SetPrecision(14);
```

Comment:
I don't think this has an effect because the default output format for ParaView data collections is binary, which is unaffected by the precision.

Reply:
Done